libr/include/rvc.h

```diff
@@ -24,6 +24,7 @@ R_API bool r_vc_commit(const char *rp, const char *message, const char *author,
 R_API bool r_vc_branch(const char *rp, const char *bname);
 R_API bool r_vc_new(const char *path);
 R_API bool r_vc_checkout(const char *rp, const char *bname);
+R_API void tree_stuff(char *rp);
```

Comment:
uhm, public apis should follow a convention, and `tree_stuff` sounds to me to something internal.

Reply:
actually this is defined and used nowhere. so its fine to be removed i guess